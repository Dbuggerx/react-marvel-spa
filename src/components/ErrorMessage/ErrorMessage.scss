@import '../scss/variables';

.error-message {
  display: flex;
  flex-flow: column nowrap;
  justify-content: center;
  height: 100vh;

  &__description {
    background: map-get($colors, primary-bg);
    color: map-get($colors, marvel-red);
    padding: 15px;
    box-sizing: border-box;
    text-align: center;
  }
}
