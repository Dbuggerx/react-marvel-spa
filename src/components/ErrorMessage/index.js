// @flow

import React from 'react';

type Props = {
    message: string,
};

const ErrorMessage = (props: Props) => (<h1>{props.message}</h1>);

ErrorMessage.displayName = 'ErrorMessage';

export default ErrorMessage;
