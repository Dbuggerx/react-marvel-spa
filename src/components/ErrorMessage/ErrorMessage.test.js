import React from 'react';
import { shallow } from 'enzyme';
import ErrorMessage from './index';

test('it renders properly', () => {
    const wrapper = shallow(<ErrorMessage message="This is a test" />);
    expect(wrapper).toMatchSnapshot();
});
