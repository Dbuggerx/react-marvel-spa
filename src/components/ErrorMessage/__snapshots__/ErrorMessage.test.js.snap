// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders properly 1`] = `
<h1>
  This is a test
</h1>
`;
